pkg/deploy/generator/templates_ci.go

```diff
@@ -0,0 +1,36 @@
+package generator
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/Azure/ARO-RP/pkg/util/arm"
+)
+
+func (g *generator) ciDevelopmentTemplate() *arm.Template {
+	t := templateStanza()
+
+	params := []string{
+		"acrName",
+		"acrLocationOverride",
+		"acrResourceId",
+	}
+
+	for _, param := range params {
+		p := &arm.TemplateParameter{Type: "string"}
+		switch param {
+		case "acrLocationOverride":
+			p.DefaultValue = "eastus"
+		case "acrName":
+			p.DefaultValue = "arosvcdev"
```

Comment:
I think our MVP ACR was `arodevsvc`? @shubhadapaithankar can you confirm (my portal auth is acting up again)

Reply:
yes , I am using `arosvcdev` ACR  from CI tenant

Reply:
Thanks for confirming Shubhada!